docs/doxygen/pages.dox

```diff
@@ -3,10 +3,10 @@
 @anchor shadow
 @brief AWS IoT Device Shadow library
 
-> A device's shadow is a JSON document that is used to store and retrieve current state information for a device. The Device Shadow service maintains a shadow for each device you connect to AWS IoT. You can use the shadow to get and set the state of a device over MQTT or HTTP, regardless of whether the device is connected to the Internet. Each device's shadow is uniquely identified by the name of the corresponding thing.
+> Device Shadows can make a device’s state available to apps and other services whether the device is connected to AWS IoT or not. AWS IoT thing objects can have multiple named shadows so that your IoT solution has more options for connecting your devices to other apps and services. AWS IoT thing objects do not have any named shadows until they are created explicitly; however, an unnamed, classic shadow is created for a thing when the thing is created.
```

Comment:
```suggestion
> Device Shadows can make a device’s state available to applications and other services whether or not the device is connected to AWS IoT. AWS IoT thing objects can have multiple named shadows so that your IoT solution has more options for connecting your devices to other applications and services. AWS IoT thing objects do not have any named shadows until they are created explicitly; however, an unnamed, classic shadow is created for a thing when the thing is created.
```

Reply:
Resolving this since it is copied directly from the docs